scss/appNav.module.scss

```diff
@@ -1,30 +1,92 @@
+@use 'sass:color';
+@use '_variables.module.scss';
+
+@media (min-width: 992px) {
+  .navbar {
+    height: 65px;
+  }
+}
+
+.navbar {
+  font-family: PT Mono;
+  box-shadow: 0px 4px 4px 0px hsla(0, 0%, 0%, 0.08);
+}
+
 .navbar-brand {
-  font-family: 'PT Mono', sans-serif;
-  font-size: 30px;
+  font-size: 40px;
 }
 
 .nav-item {
-  margin-left: 2em;
-  margin-right: 2em;
+  margin-left: 1em;
+  margin-right: 1em;
+  font-size: 14px;
+  font-weight: 700;
+  color: #565656;
+  text-transform: uppercase;
+  background-color: #fbfbfb;
+  border-radius: 4px;
+  height: 30px;
+  padding: 5px;
+}
+
+.nav-link {
+  color: #565656;
+  font-weight: 700;
+  &:hover {
+    background-color: color.change(
+      variables.$primary,
+      $lightness: 90%
+    ) !important;
+    color: variables.$primary !important;
```

Comment:
is it necessary to use `!important` in your css changes?

Reply:
yeah let's never use `!important`